public/js/controllers/game.js

```diff
@@ -147,6 +149,35 @@ angular.module('mean.system')
       if (game.state === 'waiting for czar to decide' && $scope.showTable === false) {
         $scope.showTable = true;
       }
+
+      // Check if game has ended and save the game data
+      if (game.state === 'game ended' && game.playerIndex === 0) {
+        const { players, gameID, gameWinner, round } = game;
+        const gameStarter = players[0].username;
+        const winner = players[gameWinner].username;
+        const token = localStorage.getItem('card-game-token');
+
+        $http({
+          method: 'POST',
+          url: `/api/games/${gameID}/start`,
+          data: {
+            players,
+            winner,
+            gameStarter,
+            roundsPlayed: round,
+          },
+          headers: {
+            'Content-Type': 'application/json',
+            'card-game-token': `${token}`,
+          }
+        })
+        .then(successResponse => {
+          console.log(`Game saved ${successResponse.data.game}`);
+        },
```

Comment:
Good job Amara for implementing this feature and the use of comments. Please review line 175 and save the response in a variable rather than console logging it.

Reply:
Alright, feedback noted. I will change it now.